src/harness/reference_models/fss_purge/fss_purge.py

```diff
@@ -0,0 +1,300 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""
+==================================================================================
+    This is a subset of the Pre-IAP reference model which implements
+    the FSS purge list algorithm described in R2-SGN-29.
+    The main function is fssPurgeModel().
+==================================================================================
+"""
+import numpy as np
+from reference_models.antenna import antenna
+from reference_models.interference import interference
```

Comment:
Is interference.py part of this pull request?

Reply:
This used to be included in this PR, but move out as separate one per review comments. I'll check this out where it goes to and get back to you on this. Thanks.